src/main/java/redis/clients/jedis/args/SetExpireOption.java

```diff
@@ -0,0 +1,20 @@
+package redis.clients.jedis.args;
+
+import redis.clients.jedis.util.SafeEncoder;
+import java.util.Locale;
+
+public enum SetExpireOption implements Rawable {
```

Comment:
I am not satisfied with the name `SetExpireOption`.

Off the top of my head; 1) ExpiryCondition, 2) ExpiryOption.

@dengliming @yangbodong22011 thoughts?

Reply:
`ExpiryOption` seems better. : )